src/app/components/attestation/attestation-create.jsx

```diff
@@ -121,7 +157,7 @@ function ChplAttestationCreate(props) {
           setStage(3);
         } else {
           const body = error.response.data?.error
-                || error.response.data?.errorMessages.join(' ');
+            || error.response.data?.errorMessages.join(' ');
```

Comment:
Can you see if you can figure out how to stop your editor from changing these indentations around?